austrakka/components/sequence/funcs.py

```diff
@@ -395,14 +408,34 @@ def _get_seq_api(group_name: str):
     return api_path
 
 
+def _get_seq_api_sample_names(sample_ids: List[str]):
+    api_path = SEQUENCE_PATH
```

Comment:
I think using this function will require a server-side change to make it work for non-admins. The CLI command needs to be tested using a non-admin account.

Reply:
It should be working now. I did test it and added the permission to this branch: [4718](https://github.com/AusTrakka/austrakka-portal/pull/1213)